### PR TITLE
Update per-monitor-awareness-extenders.md

### PR DESCRIPTION
**Pull request by dreddy-work**
Updating content for Winforms scenarios.

### CODE REVIEW

**Comment by PRMerger20** [comment]
@dreddy-work : Thanks for your contribution! The author, @rub8n, has been notified to review your proposed change.

**Comment by dreddy-work** [comment]
@merriemcgaw , @OliaG  please take a look at the changes made here.

File: docs/extensibility/ux-guidelines/per-monitor-awareness-extenders.md [hunk 1]
```diff
@@ -101,13 +103,18 @@ Whenever inside mixed-mode DPI scenarios (e.g. different UI elements rendering i
...
+
+#### Windows Forms controls or windows not displaying
 One of the main causes for this issue is developers trying to reparent a control or window with one DpiAwarenessContext to a window with a different DpiAwarenessContext.
 
-The following pictures show the current Windows operating system restrictions in parenting windows:
+The following pictures show the current **default** Windows operating system restrictions in parenting windows:
 
 ![A screenshot of the correct parenting behavior](../../extensibility/ux-guidelines/media/PMA-parenting-behavior.PNG)
 
+**Note:** you can change this behaviour by setting the Thread Hosting Behaviour ( refer to [DpiHostinBehaviour](https://docs.microsoft.com/en-us/windows/desktop/api/windef/ne-windef-dpi_hosting_behavior) )
```
Comment (nschonni) [inline docs/extensibility/ux-guidelines/per-monitor-awareness-extenders.md]: Docs links shouldn't have the locale in them
```suggestion
**Note:** you can change this behaviour by setting the Thread Hosting Behaviour ( refer to [DpiHostinBehaviour](https://docs.microsoft.com/windows/desktop/api/windef/ne-windef-dpi_hosting_behavior) )
```

File: docs/extensibility/ux-guidelines/per-monitor-awareness-extenders.md [hunk 2]
```diff
@@ -101,13 +103,18 @@ Whenever inside mixed-mode DPI scenarios (e.g. different UI elements rendering i
 #### Out-of-process UI
 Some UI is created out-of-process and if the creating external process is in a different DPI awareness mode than Visual Studio, this can introduce any of the previous rendering issues.
 
-#### Windows Forms controls, images, or windows not displaying
+#### Windows Forms controls, images or layouts rendered incorrectly
+Not all of the Windows Forms content support PMA mode. As a result, you may see rendering issue with incorrect layouts or scaling. A possible solution here is,  force Windows Forms content to be rendered in "System Aware"  DpiAwarenessContext ( refer to [Forcing a control into a specific DpiAwarenessContext](https://github.com/MicrosoftDocs/visualstudio-docs-pr/blob/master/docs/extensibility/ux-guidelines/per-monitor-awareness-extenders.md#forcing-a-control-into-a-specific-dpiawarenesscontext)).
```
Comment (nschonni) [inline docs/extensibility/ux-guidelines/per-monitor-awareness-extenders.md]: "visualstudio-docs-pr" isn't a public repo, so non-MS people won't be able to see this file

File: docs/extensibility/ux-guidelines/per-monitor-awareness-extenders.md (hunk 1 above)
Comment (nschonni) [reply docs/extensibility/ux-guidelines/per-monitor-awareness-extenders.md]: Might also make sense to use `[!NOTE]` alert section too

File: docs/extensibility/ux-guidelines/per-monitor-awareness-extenders.md [hunk 3]
```diff
@@ -35,11 +35,13 @@ Refer to the [High DPI Desktop Application Development on Windows](https://docs.
 ## Enabling PMA
 To enable PMA in Visual Studio, the following requirements need to be met:
 1)  Windows 10 April 2018 Update (v1803, RS4) or later
-2)  .NET Framework 4.8 RTM or greater (currently ships as standalone preview or bundle with recent Windows Insider builds)
+2)  .NET Framework 4.8 RTM or greater
 3)  Visual Studio 2019 with the ["Optimize rendering for screens with different pixel densities"](https://docs.microsoft.com/visualstudio/ide/reference/general-environment-options-dialog-box?view=vs-2019) option enabled
 
 Once these requirements are met, Visual Studio will automatically enable PMA mode across the process.
 
+**Note:** Windows Forms content in VS ( for ex: Property Browser) will support PMA only when you have Visual Studio 2019 Update #1.
```
Comment (OliaG) [inline docs/extensibility/ux-guidelines/per-monitor-awareness-extenders.md]:
```suggestion
**Note:** Windows Forms content in VS (for example Property Browser) will support PMA only when you have Visual Studio 2019 Update #1.
```

File: docs/extensibility/ux-guidelines/per-monitor-awareness-extenders.md (hunk 2 above)
Comment (OliaG) [inline docs/extensibility/ux-guidelines/per-monitor-awareness-extenders.md]:
```suggestion
Not all of the Windows Forms content support PMA mode. As a result, you may see rendering issue with incorrect layouts or scaling. A possible solution in this case is to explicitly render Windows Forms content as `SystemAware`. You can find how to set `DpiAwarenessContext` to `SystemAware`  in [Forcing a control into a specific DpiAwarenessContext](https://github.com/MicrosoftDocs/visualstudio-docs-pr/blob/master/docs/extensibility/ux-guidelines/per-monitor-awareness-extenders.md#forcing-a-control-into-a-specific-dpiawarenesscontext) paragraph of this article.
```

**Comment by PRMerger13** [comment]
@OliaG : Thanks for your contribution! The author, @rub8n, has been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@nschonni : Thanks for your contribution! The author, @rub8n, has been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@dreddy-work : Thanks for your contribution! The author, @rub8n, has been notified to review your proposed change.

File: docs/extensibility/ux-guidelines/per-monitor-awareness-extenders.md [hunk 4]
```diff
@@ -35,11 +35,13 @@ Refer to the [High DPI Desktop Application Development on Windows](https://docs.
 ## Enabling PMA
 To enable PMA in Visual Studio, the following requirements need to be met:
 1)  Windows 10 April 2018 Update (v1803, RS4) or later
-2)  .NET Framework 4.8 RTM or greater (currently ships as standalone preview or bundle with recent Windows Insider builds)
+2)  .NET Framework 4.8 RTM or greater
 3)  Visual Studio 2019 with the ["Optimize rendering for screens with different pixel densities"](https://docs.microsoft.com/visualstudio/ide/reference/general-environment-options-dialog-box?view=vs-2019) option enabled
 
 Once these requirements are met, Visual Studio will automatically enable PMA mode across the process.
 
+**Note:** Windows Forms content in VS (for example Property Browser) will support PMA only when you have Visual Studio 2019 Update #1.
```
Comment (nschonni) [inline docs/extensibility/ux-guidelines/per-monitor-awareness-extenders.md]:
```suggestion
> [!NOTE]
> Windows Forms content in VS (for example Property Browser) will support PMA only when you have Visual Studio 2019 Update #1.
```

**Comment by PRMerger12** [comment]
@nschonni : Thanks for your contribution! The author, @rub8n, has been notified to review your proposed change.

**Comment by PRMerger17** [comment]
@GitHubber17 : Thanks for your contribution! The author, @rub8n, has been notified to review your proposed change.

**Pull request merged**